FrontEnd/FrontEnd.Web/Components/Pages/Home.razor

```diff
@@ -1,8 +1,52 @@
 ﻿@page "/"
+@using FrontEnd.Web.Services
+@attribute [StreamRendering]
+@inject IRecipeService RecipeService
 
-<PageTitle>Home</PageTitle>
+<PageTitle>Recipes</PageTitle>
 
-<FluentLabel Typo="Typography.H1" MarginBlock="10px">Hello, world!</FluentLabel>
+@if (_recipes == null)
+{
+    <FluentStack Wrap="true">
+        @for (var i = 0; i < 12; ++i)
+        {
+            <FluentSkeleton Width="16rem" Height="16rem" Shimmer="true"/>
+        }
+    </FluentStack>
+}
+else if (!_recipes.Any())
+{
+    <FluentLabel Typo="Typography.Subject">No recipes exist.</FluentLabel>
+    <FluentStack VerticalAlignment="VerticalAlignment.Center" HorizontalGap="3" Style="margin-block: 10px">
+        <FluentLabel Typo="Typography.Body">Love cooking?</FluentLabel>
+        <FluentAnchor Href="recipes/new" Appearance="Appearance.Hypertext">Share your own!</FluentAnchor>
+    </FluentStack>
+}
+else
+{
+    <FluentStack Wrap="true">
+        @foreach (var recipe in _recipes)
+        {
+            <FluentNavLink Href=@($"recipes/{recipe.Id}") Match="NavLinkMatch.All">
+                <FluentCard Width="16rem" Height="16rem" Style="background-color: var(--neutral-layer-1)">
+                    <div style="height: 8rem; overflow: hidden; display: flex; justify-content: center; align-items: center;">
+                        <img src="@_recipeImageUrl" alt="@recipe.Title"/>
+                    </div>
+                    <FluentLabel Typo="Typography.Subject" MarginBlock="10px">@recipe.Title</FluentLabel>
+                    <FluentLabel Typo="Typography.Body">@recipe.Description</FluentLabel>
+                </FluentCard>
+            </FluentNavLink>
```

Comment:
Could move this out to a separate [component](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/?view=aspnetcore-8.0), e.g. `RecipeCard` or similar name